docs/pages/includes/role-spec.mdx

```diff
@@ -44,8 +44,8 @@ spec:
     # lock sets locking mode for user of this role,
     # valid values are "strict" or "best_effort"
     lock: strict
-    # enterprise-only request_access field is either 'always' or 'reason'. If set to always, it instructs
-    # tsh or the web UI clients to always create an Access Request on login. If it is
+    # enterprise-only request_access field is 'optional`, 'always' or 'reason'. If set to always or reason,
```

Comment:
```suggestion
    # enterprise-only request_access field is 'optional', 'always' or 'reason'. If set to always or reason,
```

While you're here, it would be nice if we had a standard way of referring to accepted values in this page.

Some times we single quote them (like here). Other times we double quote ("strict" or "best_effort"), and other times we use backticks.
```suggestion
    # enterprise-only request_access field is 'optional', 'always' or 'reason'. If set to always or reason,
```

Reply:
I've made this more consistent. Removed usage of ` to using ' as that's valid in the spec when used. Did leave some double quotes were seemed appropriate.